src/lib/KeyStoreIndexeddb.js

```diff
@@ -221,18 +223,16 @@ class KeyStore {
             await this.putPlain(keyEntry);
         });
 
-        // FIXME Uncomment after/for testing
+        // FIXME Uncomment after/for testing (and also adapt KeyStoreIndexeddb.spec.js)
         // await accountStore.drop();
 
         if (BrowserDetection.isIos() || BrowserDetection.isSafari()) {
             // Delete migrate cookie
-            document.cookie = 'migrate=0;expires=0';
+            document.cookie = 'migrate=0; expires=Thu, 01 Jan 1970 00:00:01 GMT;';
```

Comment:
What's wrong with `expires=0`? It works the exact same way.

Reply:
Did you try and confirm it?
MDN states that `expires=` takes a date-in-GMTString-format and the example also uses `Thu, 01 Jan 1970 00:00:01 GMT;` to delete a cookie

Reply:
> Invalid dates, like the value 0, represent a date in the past and mean that the resource is already expired.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires

Reply:
`max-age=0` also works.

Reply:
Tested in Safari and Firefox? Is it not following the standard to safe a bytes?

Reply:
The cookie is immediately deleted anyways, so I gonna follow the standard